database/exip.sql

```diff
@@ -473,6 +473,31 @@ VALUES
 UNLOCK TABLES;
 
 
+# Dump of table CoverPeriod
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `CoverPeriod`;
+
+CREATE TABLE `CoverPeriod` (
+  `id` varchar(191) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `valueId` int DEFAULT NULL,
+  `value` varchar(191) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
```

Comment:
Varchar set as `191`,  based on test below can we set it to `70` please as max characters are at around 16.

Reply:
Good shout, have added this for `CoverPeriod`and `TotalContractValue`, in the DB dump and keystone schema.